psh.proto

```diff
@@ -15,11 +15,12 @@ message PshResponse {
 
 message HostInfoRequest {
   string token = 1;
-  optional bytes ipv4 = 2;
+  optional fixed32 ipv4 = 2;
```

Comment:
Either `ipv4` is optional or `ipv6` is optional, they cannot be both optional. 

I believe `Oneof` is good for this situation.